src/main/java/org/folio/circulation/domain/validation/LoanValidator.java

```diff
@@ -13,7 +13,7 @@ private LoanValidator() {}
   public static Result<Loan> refuseWhenLoanIsClosed(Result<Loan> loanResult) {
     return loanResult.failWhen(
       loan -> succeeded(loan.isClosed()),
-      loan -> singleValidationError("Loan is closed", "id", loan.getId())
+      loan -> singleValidationError("Loan is closed", "loanId", loan.getId())
```

Comment:
In the situations where this is used, what is the incoming parameter called, is it `id` or `loanId`? Or is it part of the path for the endpoint?

Reply:
It is path parameter and called `id`, e.g. `/circulation/loans/:id/declare-claimed-returned-item-as-missing`

Reply:
Ok, thanks, let's leave as it currently is then